tests/test_graphBuilder.py

```diff
@@ -86,6 +92,30 @@ def testNoQueryNoInstruments(self):
         self.assertEqual(GraphBuilder._verifyInstrumentRestriction("", ""), "")
 
 
+class GraphBuilderTestCase(unittest.TestCase):
+
+    def testDefault(self):
+        """Simple test to verify makeSimpleQGraph can be used to make a Quantum
+        Graph."""
+        with tempfile.TemporaryDirectory() as root:
```

Comment:
Tim pointed out to me that `tempfile.TemporaryDirectory` may not behaving well if there are errors during deleting the directory, it does not specify `ignore_errors=True` when it calls `shutil.rmtree`. I had to implement trivial context manager to workaround that: https://github.com/lsst/ctrl_mpexec/blob/master/tests/test_cmdLineFwk.py#L78-L87. I think you need something like that too (better of course is to have it somewhere in the common code base but I don't know where that could be). Same for the method below.